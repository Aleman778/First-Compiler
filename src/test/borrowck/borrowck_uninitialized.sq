// ignore

fn main() {
    let x: i32;
    borrow(&x);
}


fn borrow(x: &i32) -> i32 {
    *x + 10
}
