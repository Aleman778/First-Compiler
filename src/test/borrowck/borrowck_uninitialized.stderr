error: borrow of possibly uninitialized variable: `x`
  --> $DIR/borrowck_uninitialized.sq:5:12
   |
LL |     borrow(&x);
   |            ^^ use of possibly uninitialized `x`
